keel-core/src/main/kotlin/com/netflix/spinnaker/keel/telemetry/TelemetryEvent.kt

```diff
@@ -121,25 +121,28 @@ data class VerificationCompleted(
   val application: String,
   val deliveryConfigName: String,
   val environmentName: String,
-  val artifactName: String,
+  val artifactReference: String,
```

Comment:
@lorin @robfletcher we should always be using reference and not name. Nothing was using this event, so I changed it here. But a note for the future.